src/scripts/backupScript.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# 
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+# Input the pathname for the desired backup directory
+# ie PATH="/home/<your_username>/path_to/database_dumps/dump.sql"
```

Comment:
This isn't really an explicit path but a file location including the path. Maybe update the comment. Maybe also note that it will be the file listed but with the date and give an example.

Also, if the directory does not exist then this fails. At a minimum there should be a comment if that is the case. An alternative is to check if there it exists and create if it does not. Helpful for the first time this runs.

Reply:
Added an if statement that checks to see if the file path exists, and lets the user know if it exists. If not, it attempts to create the database_dumps directory. If this is unsuccessful, a message will then be displayed saying it failed. Thank you for bringing this up.